ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -252,8 +278,8 @@ private void initializeTable() {
 
         logger.debug("Creating table if it does not exist yet");
 
-        Long readCapacityUnits = Long.parseLong(this.config.getReadCapacityUnits());
-        Long writeCapacityUnits = Long.parseLong(this.config.getWriteCapacityUnits());
+        Long readCapacityUnits = this.config.getReadCapacityUnits();
```

Comment:
Note for myself: In my initial PR for the DDB plugin https://github.com/Netflix/ndbench/pull/76 I had this as `Long` and Eureka had some issues with it. That is why I changed it to String and did the funky parseLong stuff.

Reply:
fixed